nf_core/components/components_command.py

```diff
@@ -48,24 +59,25 @@ def _configure_repo_and_paths(self, nf_dir_req=True):
                 raise
             self.repo_type = None
             self.org = ""
-        self.default_modules_path = Path("modules", self.org)
-        self.default_tests_path = Path("tests", "modules", self.org)
-        self.default_subworkflows_path = Path("subworkflows", self.org)
-        self.default_subworkflows_tests_path = Path("tests", "subworkflows", self.org)
+        self.default_modules_path: Path = Path("modules", self.org)
+        self.default_tests_path: Path = Path("tests", "modules", self.org)
+        self.default_subworkflows_path: Path = Path("subworkflows", self.org)
+        self.default_subworkflows_tests_path: Path = Path("tests", "subworkflows", self.org)
 
-    def get_local_components(self):
+    def get_local_components(self) -> List[str]:
         """
         Get the local modules/subworkflows in a pipeline
         """
-        local_component_dir = Path(self.dir, self.component_type, "local")
+        local_component_dir: Path = Path(self.dir, self.component_type, "local")
```

Comment:
I think this is not needed
```suggestion
        local_component_dir = Path(self.dir, self.component_type, "local")
```